src/components/base/Base.js

```diff
@@ -395,28 +395,29 @@ export class BaseComponent {
   }
 
   setupValueElement(element) {
-    let value = this.value;
+    let value = this.getValue();
     value = this.isEmpty(value) ? this.defaultViewOnlyValue : this.getView(value);
-    element.appendChild(this.text(value));
+    element.innerHTML = value;
   }
 
   get defaultViewOnlyValue() {
     return '-';
   }
 
   getView(value) {
-    return _.toString(value);
-  }
+    if (Array.isArray(value)) {
+      return value.join(', ');
+    }
 
-  updateViewOnlyValue() {
-    this.empty(this.valueElement);
-    this.setupValueElement(this.valueElement);
+    return value.toString();
   }
 
-  empty(element) {
```

Comment:
Please don't remove this. It is a helpful helper method that I am using in the form builder branch.